lib/controllers/frontend/spree/user_registrations_controller.rb

```diff
@@ -30,6 +30,7 @@ def new
   # POST /resource/sign_up
   def create
     @user = build_resource(spree_user_params)
+    @user.saved_locale = current_locale
```

Comment:
Have you looked into including locale in `spree_user_params` so that it's part of the collection of values used to build the user registration? Would be cleaner that way.

Reply:
I've moved adding locale to function spree_user_params.